spec/commands/config_spec.rb

```diff
@@ -187,6 +187,16 @@
     end
   end
 
+  context "with --parseable option" do
+    it "returns value associated with the config" do
```

Comment:
please add a test for when the value is unset, and also when the value is unset but bundler has a default value